meson.build

```diff
@@ -110,6 +110,10 @@ client_protos = declare_dependency(
 conf_data = configuration_data()
 conf_data.set10('HAVE_GDK_PIXBUF', gdk_pixbuf.found())
 
+if cc.has_function('explicit_bzero')
```

Comment:
Maybe add `prefix: "#include <string.h>"`?

Reply:
Im new to meson build system. Could you add this yourself please?

Reply:
```meson
explicit_bzero_prefix = '''#include <string.h>
#include <strings.h>'''
cc.has_function('explicit_bzero', prefix: explicit_bzero_prefix)
```

Reply:
Doesn't work. If done like this, meson will compile `defined(explicit_bzero)` to check for that fuction, but checking for functions with defined() is wrong, it will always return False if that symbol (function) isn't (also) a macro.

Reply:
In my case, it is't also a macro, and thus the detection fails.

Reply:
Hmm, it sounds like the issue is that `explicit_bzero` requires `GNU_SOURCE` or `BSD_SOURCE` on glibc/musl?

And on BSDs it requires _not_ setting `_POSIX_C_SOURCE`?

Reply:
Added the missing args : '-D_GNU_SOURCE', works under linux glibc, should also work under musl. untested unter *bsd.